### ISSUE TITLE
Problem with Database

### ISSUE THREAD

**Issue by Rudi1986** [phiilu/mailman#37]
Hi,

i just made an update and after it Mailman won't let me log in anymore.
mailman-error-1.log says: "Error: No database selected". After playing around with .env (new Variable) i took a closer lock at your last changes in db.js. Especially the line where the DB is selected.
Old:    db: process.env.MAILMAN_DB_DATABASE
New:    database: process.env.MAILMAN_DB_DATABASE

I changed this line back to the old version. And it works just fine.

Greetings from Munich
Rudi

**Comment by hibas123** [comment]
Did you run `npm install` after the update?

**Comment by Rudi1986** [comment]
Yes.

**Comment by phiilu** [comment]
You might be right, that this will cause a problem. I merged this [PR](https://github.com/phiilu/mailman/pull/36/files#diff-f786454c0651c074b51fa96e1b531414L9) and indeed this line was changed. 

I think in a newer version of knex (the database client) the field is called database instead of db. 

I am working on a newer version of Mailman and hope I can release it till the end of this year. Not so easy to find time anymore.
